Modules/Core/Common/test/itkImageReverseIteratorTest.cxx

```diff
@@ -98,9 +98,6 @@ itkImageReverseIteratorTest(int, char *[])
   (*o3)[regionEndIndex3D] = (*o3)[regionStartIndex3D];
   TestConstPixelAccess(*o3, *o3);
 
-
-  itk::ImageIterator<ImageType> standardIt(o3, region);
```

Comment:
Again, nice catch. This `standardIt` variable also appears unused for a long time. Already with its introduction, September 30, 2002:

https://github.com/InsightSoftwareConsortium/ITK/blob/2cb06aafc450c5afc30e792192c23d44b1307fa1/Testing/Code/Common/itkImageReverseIteratorTest.cxx#L96

Reply:
I found it reading the code for other reasons.